analyses/molecular-subtyping-HGG/02-HGG-molecular-subtyping-subset-files.R

```diff
@@ -138,7 +138,11 @@ tumor_metadata_df <- metadata %>%
 # Samples included on the basis of the pathology diagnosis fields
 path_dx_df <- tumor_metadata_df %>%
   # Inclusion on the basis of CBTTC harmonized pathology diagnoses
-  filter(pathology_diagnosis %in% path_dx_list$exact_path_dx)
+  filter(pathology_diagnosis %in% path_dx_list$exact_path_dx |
+         # Inclusion based on pathology free text diagnosis
+         pathology_free_text_diagnosis ==path_dx_list$gliomatosis_path_free_text_exact)
```

Comment:
Checking my understanding - the removal of "Gliomatosis Cerebri" from the `exact_path_dx`  is what resulted in the removal `7316-3817` and we add `gliomatosis_path_free_text_exact` to capture gliomatosis cerebri samples that we should specifically include in HGG subtyping, correct?

Reply:
correct - it will be removed for v18, but in v19 will be added back due to the `pathology_diagnosis` being updated to HGG per [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/972#issuecomment-810428414), but this ensures we do not capture any LGG GC samples when we do not have grades.